### PR TITLE
FAT2-88 sectors

### PR DESCRIPTION
**Pull request by dashton82**

### CODE REVIEW

File: src/SFA.DAS.FAT.Web.UnitTests/Models/WhenCreatingCoursesViewModel.cs [hunk 1]
```diff
@@ -23,5 +29,71 @@ public void Then_The_Total_Message_Is_Created_Correctly(int totalCount,int filte
...
+                
+                Assert.IsNotNull(sector);
+                Assert.IsTrue(model.ClearSectorLinks.ContainsKey(sector.Route));
+                Assert.AreEqual(clearLinkCount-1, model.ClearSectorLinks.Count(c=>c.Value.Contains($"sectors={selectedRoute}")));    
+                Assert.AreEqual(clearLinkCount, model.ClearSectorLinks.Count(c=>c.Value.Contains($"keyword={keyword}")));    
+            }
+
+            
+        }
+
```
Comment (scottywakefield) [inline src/SFA.DAS.FAT.Web.UnitTests/Models/WhenCreatingCoursesViewModel.cs:77]: I think there's another test for keyword but no sectors.

File: src/SFA.DAS.FAT.Web/Models/CoursesViewModel.cs [hunk 2]
```diff
@@ -1,22 +1,61 @@
...
+        public int TotalFiltered { get ; set ; }
+        public List<SectorViewModel> Sectors { get ; set ; }
+        public string TotalMessage => GetTotalMessage();
+        public List<Guid> SelectedSectors { get ; set ; }
+        public Dictionary<string, string> ClearSectorLinks => BuildClearSelectedFilterLink();
+        public string ClearKeywordLink => BuildClearKeywordFilterLink();
+        
         private string GetTotalMessage()
         {
             var totalToUse = string.IsNullOrEmpty(Keyword) ? Total : TotalFiltered;
```
Comment (scottywakefield) [inline src/SFA.DAS.FAT.Web/Models/CoursesViewModel.cs:21]: this is different now that sectors might have reduced the total right?

File: src/SFA.DAS.FAT.Web.UnitTests/Models/WhenCreatingCoursesViewModel.cs (hunk 1 above)
Comment (dashton82) [reply src/SFA.DAS.FAT.Web.UnitTests/Models/WhenCreatingCoursesViewModel.cs:77]: Added test to cover this

File: src/SFA.DAS.FAT.Web/Models/CoursesViewModel.cs (hunk 2 above)
Comment (dashton82) [reply src/SFA.DAS.FAT.Web/Models/CoursesViewModel.cs]: Yeah missed that one - added test and updated.

**Pull request merged**